packaging/desktop/package_darwin.sh

```diff
@@ -138,6 +138,13 @@ get_deps() {(
   curl -J -L -Ss "$installer_url" | tar zx
 )}
 
+check_ci() {(
+  temp="$(mktemp -d)"
+  (cd "$temp" && npm i github-ci-status)
```

Comment:
to clarify, since this is confusing, there are two npm installs in this script:

- `keybase/client/packaging/desktop/package.json`: For the build script
- `keybase/client/desktop/package.json`: For the Electron app